src/tests/mocks/erc721_mocks.cairo

```diff
@@ -146,48 +146,42 @@ mod CamelERC721Mock {
         SRC5Event: SRC5Component::Event
     }
 
-    #[constructor]
-    fn constructor(
-        ref self: ContractState,
-        name: felt252,
-        symbol: felt252,
-        recipient: ContractAddress,
-        token_id: u256,
-        uri: felt252
-    ) {
-        self.erc721.initializer(name, symbol);
-        self.erc721._mint(recipient, token_id);
-        self.erc721._set_token_uri(token_id, uri);
-    }
-
     /// The following external methods are included because they are case-agnostic
     /// and this contract should not embed the snake_case impl.
-    #[generate_trait]
     #[abi(per_item)]
+    #[generate_trait]
     impl ExternalImpl of ExternalTrait {
+        #[constructor]
```

Comment:
Following up on the previous comment, I don't think the constructor belongs to an external trait, the constructor is not an external method.